### PR TITLE
Translations for decodeJwt function

### PR DESCRIPTION
**Pull request by baltpeter**
In preparation for: https://github.com/tweaselORG/TrackHAR/pull/81

This can be merged before the changes in TrackHAR are released.

### CODE REVIEW

File: i18n/en.json [hunk 1]
```diff
@@ -64,6 +64,10 @@
         "one": "Decode {{ .input }} as a protocol buffer (Protobuf).",
         "other": "Decode every element in {{ .mapInput }} as a protocol buffer (Protobuf)."
     },
+    "decoding-step-function-decodeJwt": {
+        "one": "Decode {{ .input }} as a JSON Web Token (JWT).",
+        "other": "Decode every element in {{ .mapInput }} as a JSON Web Token (JWT)."
```
Comment (zner0L) [inline i18n/en.json:69]:
```suggestion
        "one": "Decode {{ .input }} from a JSON Web Token (JWT).",
        "other": "Decode every element in {{ .mapInput }} from a JSON Web Token (JWT)."
```

I am not sure which preposition fits, but ‘as’ feels wrong.

Comment (zner0L) [reply i18n/en.json:69]: But this is what "from" canonically is supposed to mean, see e.g. https://www.urldecoder.org/

Comment (baltpeter) [reply i18n/en.json:69]: Setting aside that I wouldn't accept a random website as "canonical" (:P), you decode the _output_ (unencoded message) from [an encoding], not the input (encoded message) as in your suggested phrasing.

Unlike our sentence, your canonical source doesn't have a subject in any sentence where it uses "decode from".

Comment (baltpeter) [reply i18n/en.json:69]: But in any case, this discussion is not specific to JWT. We are using "as" in all the other decode function translations as well.

Comment (zner0L) [reply i18n/en.json:69]: I think maybe we should remove the prepostions altogether.

Comment (baltpeter) [reply i18n/en.json:69]: Huh, what would that look like? But I genuinely don't see the issue you have with "as".

**Review by zner0L — APPROVED** [review]

**Pull request merged**